app/src/main/java/ch/deletescape/lawnchair/PagedView.java

```diff
@@ -1425,6 +1386,29 @@ public int getScrollForPage(int index) {
         }
     }
 
+    // This curve determines how the effect of scrolling over the limits of the page dimishes
+    // as the user pulls further and further from the bounds
+    private float overScrollInfluenceCurve(float f) {
+        f -= 1.0f;
+        return f * f * f + 1.0f;
+    }
+
+    protected float acceleratedOverFactor(float amount) {
```

Comment:
We also might wanna check if the device is in battery save mode to disable spring animation.

Reply:
Actually this is just damping the scroll distance, not as demanding as the spring animation.